package.json

```diff
@@ -156,14 +156,15 @@
     "piping": "0.3.2",
     "react": "15.3.1",
     "react-addons-css-transition-group": "15.3.1",
+    "redux-async-connect": "1.0.0-rc4",
```

Comment:
Though in this line's case it's just alphabetising. I don't recall doing that manually, but I'll make sure it's not changing any versions and push a fix.